camel/toolkits/hybrid_browser_toolkit/ts/src/browser-session.ts

```
@@ -0,0 +1,860 @@
+    
+    try {
+      const elementSearchStart = Date.now();
+      
+      //  No need to pre-fetch snapshot - each action method handles this
+      
+      let newTabId: string | undefined;
+      
+      switch (action.type) {
+        case 'click':
+          elementSearchTime = Date.now() - elementSearchStart;
+          const clickStart = Date.now();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)